wiki/en/en-Installation-for-Windows.md

```diff
@@ -9,7 +9,7 @@ permalink: "/wiki/Installation-for-Windows"
 
 # Installation for Windows
 Make sure you read the [Getting Started](Getting-Started) page.
-1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install [this free ASIO driver (ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} before installing Jamulus.
+1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install a generic driver like [ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"}. For more information scroll down to the [ASIO](#asio) section.
```

Comment:
I wonder if we could make it even clearer where to click on the ASIO4ALL website.
- It seems like there is no dedicated download page and one has to scroll down quite a bit as there are recent news about an Intel-SST-related issue.
- The latest version (2.14) is only available in English, so people go for the latest version available in their language (2.13). However, 2.13 lacks an important bugfix. So I wonder if we should document it as "Get at least version 2.14"?
- I was shocked when I saw someone of my choir opening the ASIO4ALL site without an ad blocker. There are lots of ads and some of them are **really** misleading, if not malicious ("Driver download" in-page popup). A friend of mine even somehow ended up on a site which wanted him to enter his credit card details. I wonder if we should warn about that, too. That's a problem in SourceForge, too, btw.

```suggestion
1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install a generic driver like [ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"}. Get at least version 2.14, even if is not available in your language. The proper download is called "ASIO4ALL 2.14 – English" (or newer) and has a flag next to it. ASIO4ALL is free. If you are asked for your credit card details or any other personal data, you have likely clicked on the wrong link. For more information scroll down to the [ASIO](#asio) section.
```

I'm unsure how many details we should add (i.e. specific versions), but this is a common preparation step which is required by many (often unexperienced) users, I feel somehow responsible for protecting our usres.

Reply:
> The latest version (2.14) is only available in English, so people go for the latest version available in their language (2.13). However, 2.13 lacks an important bugfix. So I wonder if we should document it as "Get at least version 2.14"?

For me it was the exact opposite. V 2.14 was less stable than v 2.13. That's why I usually suggest v 2.13.

> I was shocked when I saw someone of my choir opening the ASIO4ALL site without an ad blocker.

Interesting fact. I didn't know that there were ads? Are they part of the new page too?

I agree that the download link is hidden.

Reply:
> > The latest version (2.14) is only available in English, so people go for the latest version available in their language (2.13). However, 2.13 lacks an important bugfix. So I wonder if we should document it as "Get at least version 2.14"?
> 
> For me it was the exact opposite. V 2.14 was less stable than v 2.13. That's why I usually suggest v 2.13.

Interesting, then we should probably keep it as-is and don't make a version suggestion.

> > I was shocked when I saw someone of my choir opening the ASIO4ALL site without an ad blocker.
> 
> Interesting fact. I didn't know that there were ads? Are they part of the new page too?

Didn't know about the ads either, guess, most technical people live in an ad-blocked filter bubble. :)
No idea if this is new or not. 
![asio4all](https://user-images.githubusercontent.com/762915/108711759-1e1ed700-7516-11eb-9f30-a6a4cb948c4f.png)
Attaching a screenshot for reference.

Reply:
Wow so many ads! Tempting to host our own download link and just update that from time to time ... :-)

Reply:
@gilgongo should we include this in the next release of the website too?

Reply:
@ann0see I'm not sure what the implications of hosting the installer ourselves would be, if that's what you mean. I would certainly be a better experience though!

Reply:
The author clearly asserts copyright and grants no redistribution licence.  So we can't.

Reply:
I think the license shown in the installer is a bit different. I think it allows redistribution

Reply:
Here's the relevant quote (v2.14 - I don't have 2.13 to hand) - you could be right:

> YOU MAY FURTHER REDISTRIBUTE ONLY VERBATIM COPIES OF THIS SOFTWARE WITH THE FOLLOWING RESTRICTIONS:
>
> 1.) THE SOLE SOURCE OF PROFIT TO BE MADE (IF ANY) - INCLUDING PROFIT INTENTED TO BE MADE - FROM THIS SOFTWARE SHALL BE THE SERVICE OF ITS REDISTRIBUTION ONLY. THIS SPECIFICALLY PROHIBITS PLACING SOFTWARE ON ANY MEDIA SOLD TOGETHER WITH AUDIO HARD- AND SOFTWARE, OR SIMILAR PRODUCTS TO WHICH THIS SOFTWARE WOULD ADD SUBSTANTIAL MARKET VALUE.
> 2.) EXCEPTIONS TO 1. REQUIRE WRITTEN CONSENT FROM THE AUTHOR.

We don't intend to profit at all from Jamulus, not even from redistribution.  The "specifically prohibts" would kick in if we did.

Anyone wanting to distribute Jamulus for profit would have to do it elsewhere.

Reply:
OK so that does look like we can host the download ourselves then. It then only remains for us to keep an eye on version updates I suppose.

Reply:
Sounds good. One could even think about installing it from the Jamulus installer?
Any volunteers for that? Should probably be its own ticket by now :)

Reply:
I'd create a repo where we host the ASIO4ALL installer for now. I wouldn't include it on the website repo since it might make the repo too large.

Reply:
452KB for the 2.14EN.  It's not that big :).  What's the repo limit again?

Reply:
>  It's not that big :)

No. But I don't really like the idea of including binaries in this repo. I'd suggest to just host it on another repo (jamulussoftware/assets) since it would allow us to just push new releases easily.

Reply:
![image](https://user-images.githubusercontent.com/1549463/108885547-de341e80-75ff-11eb-8e1e-b7d93855c82d.png)
There's a lot of binaries, some quite a bit bigger...  But yes - anything we expect will "never change" by the project itself shouldn't live here, I'd agree on that basis! :)

Reply:
Ok. I'll set up another repo for this binary.